dev-docs/bidders/ix.md

```diff
@@ -272,47 +263,75 @@ ortb2Imp: {
 **Note:** When you use the Index renderer for outstream video, all impressions are considered viewable, which is similar to how Google's ActiveView counts impressions for outstream. This is because Index renders the outstream video as soon as it is in view and concurrently fires any impression pixels in the VAST.
 
 To use Index’s outstream video player, in your Prebid.js configuration:<br />
 
-1. Define a new video object in any of the following ways:
-    * At the `adUnit` level: Define the size using `video.playerSize`.
-    * At the `bidder` level: Define the size of the video player using the `video.h` and `video.w` parameters.  <br />
+1. Perform the following steps in your Prebid.js configuration to create a new section for Index's outstream video player:
+    * Add a new code property under `adUnits`. The code could be the `divID` or the Google Ad Manager adUnit code.
+    * Within the `adUnits`, add `ix` as a `bidder`.
+    * (Recommended) At the `adUnit` level, under `mediaTypes`, add a `video` object that supports our required video parameters. For more information about which parameters to add, see the [Bid request parameters](#video) section below. This is useful for publishers who want to apply the same settings across all SSPs.
+    * Optionally, you can add the `video` object at the bidder level under `bids.params`. This is useful for publishers who may want to set up different configurations for different SSPs.   <br />
 **Note:** The `bidder` level video configurations override the `adUnit` level configurations. The `playerConfig` is only a bidder level configuration.
-2. Configure the player according to the options in the [Bid request parameters](#bid-request-parameters) section below. <br />
+2. (Recommended) To force the Index renderer as the default renderer for Index, set `backupOnly` to true (see below example). This configuration allows you to use your own renderer only when an adapter does not provide their own renderer. This also forces any adapter that has a renderer to use their renderer by default. If no renderer is provided, by default,  we will use our own renderer script.
+3. Configure the player according to the options in the [Bid request parameters](#bid-request-parameters) section below.
+4. Depending on your existing Prebid setup, complete one of the following:
+    * If you have an existing Prebid.js integration for banner, you can use the corresponding line items in your Google Ad Manager (GAM) account to retrieve outstream video demand. For more information, see Prebid's documentation on [How to set up line items](https://docs.prebid.org/adops/step-by-step.html).
```

Comment:
Please update links to docs.prebid.org to be relative. i.e. in this case /adops/step-by-step.html

Reply:
done and will follow the same pattern going forward